scan/src/components/ChainInfoHighlights.re

```diff
@@ -134,7 +134,8 @@ let make = (~latestBlockSub: Sub.t(BlockSub.t)) => {
         let%Sub (_, {financial}, _) = allSub;
         (
           {
-            let marketcap = "$" ++ financial.usdMarketCap->Format.fPretty;
+            let marketcap =
+              "$" ++ (financial.usdMarketCap /. 1e6 |> Format.fPretty(~digits=1)) ++ "M";
```

Comment:
Let's make the helper function in `Format.re` to handle.
There are 3 conditions.
1. value >= 1e9, return in B
2. value >= 1e6, return in M
3. value >= 1e3, return in K